pom.xml

```diff
@@ -13,7 +13,6 @@
     <version>${changelist}</version>
 
     <name>Jenkins Job Configuration History Plugin</name>
-    <description>Saves copies of job, node and system configurations.</description>
```

Comment:
@NotMyFault why did you remove this line?

Reply:
To comply with the standards of an upcoming hard requirement: https://github.com/jenkinsci/maven-hpi-plugin/blob/master/src/main/java/org/jenkinsci/maven/plugins/hpi/HpiMojo.java#L139